files/sysdig-inline-scan.sh

```diff
@@ -553,23 +551,46 @@ display_report() {
 
     status=$(jq -r ".[0][][][0].status // empty" "${TMP_PATH}"/sysdig_report.log)
 
-    if [[ -z "${json_flag:-}" ]]; then
-        print_info "Scan Report:"
-        print_info_pipe < "${TMP_PATH}"/sysdig_report.log
+    print_info ""
+    print_info "Scan Report"
+
+    if [[ "${v_flag:-}" ]]; then
+        print_info_pipe "  " < "${TMP_PATH}"/sysdig_report.log
     fi
 
+    # shellcheck disable=SC2016
+    JQ_FORMAT=(
+        '.[0][][][0].detail.result.image_id as $image_id |'
+        '"\n" +'
+        '"Last Evaluation: " + .[0][][][0].last_evaluation + "\n" +'
+        '"Final action:    " + .[0][][][0].detail.result.result[$image_id].result.final_action + "\n" +'
+        '"Status:          " + .[0][][][0].status + "\n\n" +'
+        '"Evaluation results\n" '
+    )
+    print_info "$(jq -r "${JQ_FORMAT[*]}" "${TMP_PATH}"/sysdig_report.log)"
+
+    # shellcheck disable=SC2016
+    JQ_FORMAT=(
+        '.[0][][][0].detail.result.image_id as $image_id | '
+        '.[0][][][0].detail.result.result[$image_id].result.header as $headers | '
+        '($headers|index("Gate_Action")) as $action_col | '
+        '($headers|index("Gate")) as $gate_col | '
+        '($headers|index("Trigger")) as $trigger_col | '
+        '($headers|index("Check_Output")) as $output_col | '
+        '(.[0][][][0].detail.result.result[$image_id].result.rows[] | " - " + .[$action_col] + " " + .[$gate_col] + ":" + .[$trigger_col] + " " + .[$output_col])'
+    )
+    print_info "$(jq -r "${JQ_FORMAT[*]}" "${TMP_PATH}"/sysdig_report.log)"
```

Comment:
What happens if the API response is slightly different? Does `jq` exits with error and thus the entire script execution fails?
I think we should avoid failing in this case, since it's mostly a formatting issue.

Reply:
Agree. I just added error handling for all the cases where we use jq to parse the scan report, and a test.

Reply:
👍  thank you!